requirements.txt

```diff
@@ -1,9 +1,9 @@
-pybtex==0.20.1
-Pygments>=2.0,<3
-Sphinx==1.3.6
-sphinx_rtd_theme==0.1.9
-sphinxcontrib-bibtex==0.3.3
-numpy==1.11.0
-matplotlib==1.5.1
+pybtex
```

Comment:
These packages are developed independently of python.  They each have their own website with information on each release, and I would guess the numbering lining up is correlation rather than anything else.